client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/index.tsx

```diff
@@ -21,6 +23,15 @@ const Launchpad: Step = ( { navigation } ) => {
 		</div>
 	);
 
+	const site = useSite();
+	const launchpadScreenOption = site?.options?.launchpad_screen;
+
+	useEffect( () => {
+		if ( launchpadScreenOption === 'off' ) {
+			window.location.replace( `/view/${ siteSlug }` );
+		}
```

Comment:
If Launchpad is off, why not redirect to MyHome? Is there more context for why we are redirecting to 'view' here?

Reply:
I'm going to update this logic to navigate to `/home` instead as we discussed in the team meeting.